lib/task/jobs/jobWorkerTask.class.php

```diff
@@ -67,6 +67,39 @@ public function getJobsCompleted()
         return $this->jobsCompleted;
     }
 
+    public static function getMemoryUsageString()
+    {
+        return sprintf(
+            "Worker memory usage (PHP - VmRSS): %.2fMb - %dkb\n",
```

Comment:
I think it would be better to standardize the units here, rather than mixing Mb and kb.

Reply:
Good idea - done